DigitalLearningSolutions.Data.Migrations/202308011145_AddRoleLimitColumnsToCentresTable.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace DigitalLearningSolutions.Data.Migrations
+{
+    using FluentMigrator;
+    
+    [Migration(202308011145)]
+    public class AddRoleLimitColumnsToCentresTable : Migration
+    {
+        public override void Up()
+        {
+            Alter.Table("Centres").AddColumn("RoleLimitCMSAdministrators").AsInt32().Nullable().WithDefaultValue(null);
```

Comment:
These fields already exist in the Centres table (though not as well named):

- CMSAdministrators
- CMSManagers
- CustomCourses
- CCLicences
- Trainers
We should be managing the existing fields rather than introducing new ones, since these are referenced elsewhere in the application.

Reply:
Apologies- I thought those represented the 'current' values and we were adding limits.